### PR TITLE
RF-11436 - Create cop for suggesting people use API::Base and not Grape::API

### PR DESCRIPTION
**Pull request by shanempope**

### CODE REVIEW

**Comment by jbarber** [comment]
I think we should start getting the rest of the backend team to review this as part of the process, so they have a voice in what we're making rubocop complain about. This isn't specific to this particular rule, but just a process thing I just realized we should start doing.

**Comment by jbarber** [comment]
@rainforestapp/backend - please give this a look and see if you agree with whether we should add this as a linting rule or not. Perhaps we need a better process about discussing this than in the PR (in a tech plan?) Thoughts please!

File: lib/rf/stylez.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 require "rf/stylez/version"
 require 'rubocop/cop/lint/no_env'
 require 'rubocop/cop/lint/no_http_party'
+require 'rubocop/cop/lint/obscure'
```
Comment (jbarber) [inline lib/rf/stylez.rb]: :man_facepalming:

File: spec/rubocop/cop/lint/no_http_party_spec.rb [hunk 2]
```diff
@@ -4,14 +4,14 @@
   let(:config) { RuboCop::Config.new }
   subject(:cop) { described_class.new(config) }
 
-  it 'registers an offense when using `#ENV`' do
+  it 'registers an offense when using `HTTParty`' do
```
Comment (jbarber) [inline spec/rubocop/cop/lint/no_http_party_spec.rb]: :man_facepalming:

**Comment by jaytennier** [comment]
Adding this cop makes sense to me 👍

**Comment by epaulet** [comment]
This makes sense to me as well.

**Pull request merged**